addon/validations/factory.js

```diff
@@ -407,6 +408,19 @@ function createAttrsClass(validatableAttributes, validationRules, model) {
  */
 function createCPValidationFor(attribute, model, validations) {
   let isVolatile = hasOption(validations, 'volatile', true);
+
+  deprecate(
+    '[ember-cp-validations] The `volatile` option should no longer be used ' +
+      'as it was removed in ember 4.0',
```

Comment:
Should probably suggest an alternative. Maybe something like: Instead you can use a getter to accomplish the same thing.

Reply:
The problem is that all validators are computed properties. To accomplish the same thing you would need to have a dependent key that is for example a timestamp that always changes...

Reply:
Yeah, you are right. Been a while since I used them.